movement/watch_faces/settings/set_time_face.c

```diff
@@ -97,7 +97,9 @@ bool set_time_face_loop(movement_event_t event, movement_settings_t *settings, v
 
     switch (event.event_type) {
         case EVENT_TICK:
-            if (_quick_ticks_running) _handle_alarm_button(settings, date_time, current_page);
+            if (_quick_ticks_running) 
+                if (watch_get_pin_level(BTN_ALARM)) _handle_alarm_button(settings, date_time, current_page);
+                else _abort_quick_ticks();
```

Comment:
Appreciate this stopgap! I'll test some more this weekend in the hopes of merging it in; I'd love to fix the underlying issue, but after a couple of hours of chasing it today I'm pretty flummoxed, and as workarounds go, this is pretty minimal :)

My only thought would be to wrap the inner if/else in braces; the indent feels clear to me, but if someone comes in to add something later, it could lead to confusion or glitchiness (cf. [goto fail](https://dwheeler.com/essays/apple-goto-fail.html))

Reply:
You are right regarding the missing braces. I saw this yesterday, too, and have fixed it in the last commit of this PR.